### PR TITLE
Database

### PR DESCRIPTION
**Pull request by sampler36**

### CODE REVIEW

File: database/migrations/20190625083341_createArtistTable.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('artist', (artist) => {
```
Comment (maximesalomon) [inline database/migrations/20190625083341_createArtistTable.js]: Check the artist object we talked about this morning = https://lambdaschoolstudents.slack.com/archives/GKSFUFP5W/p1561447427001200

artist: {
        id: '',
        artistName: '',
        userDescription: '',
        profilePictureUrl:'',
        username: '',
        email: '',
        password: '',
    }

**Review by maximesalomon — COMMENTED** [review]
Hello Antonio 👋 

Seems like there are differences between the artist we agreed on and the one you created here.

Can you fix it?

Thanks 🙏

File: database/migrations/20190625083341_createArtistTable.js (hunk 1 above)
Comment (sampler36) [reply database/migrations/20190625083341_createArtistTable.js]: resolved

**Review by maximesalomon — COMMENTED** [review]
Good job :)

**Pull request merged**